testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/HiveProductTest.java

```diff
@@ -15,10 +15,15 @@
 
 import com.google.inject.Inject;
 import io.trino.tempto.ProductTest;
+import io.trino.tests.product.TestGroups;
 
 public class HiveProductTest
         extends ProductTest
 {
+    static {
+        TestGroups.FakeUsageForMavenDependencyChecker.fakeUse();
```

Comment:
nit: comment why needed

Reply:
class name is "fake usage for maven dependency checker". that's what i would write in a code comment. should i add more context?